modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/display/view.jsp

```diff
@@ -197,7 +197,7 @@ long formInstanceId = ddmFormDisplayContext.getFormInstanceId();
 						Liferay.fire(
 							'ddmFormView',
 							{
-								formId: <%= formInstanceId %>,
+								formId: '<%= formInstanceId %>',
```

Comment:
@interaminense, do you need to get this value as a `String` from now on?

Reply:
Yes, this is a request from the backend to receive IDs in `String` format.